sesame/src/main/scala/org/w3/banana/sesame/SesameOps.scala

```diff
@@ -167,7 +167,7 @@ class SesameOps extends RDFOps[Sesame] with SesameMGraphOps with DefaultURIOps[S
   def isomorphism(left: Sesame#Graph, right: Sesame#Graph): Boolean = {
     val leftNoContext = left.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
     val rightNoContext = right.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
-    ModelUtil.equals(leftNoContext, rightNoContext)
+    Models.isomorphic(leftNoContext, rightNoContext)
```

Comment:
Ok. So in that case, is the workaround still needed? Is the comment still accurate? Can you please check?

This is *pretty important* as this touches the core implementation of the RDFOps typeclass, which is supposed to be as close as possible to the native implementation.